src/Serilog.Sinks.AzureDataExplorer.Samples/Serilog.Sinks.AzureDataExplorer.Samples.csproj

```diff
@@ -8,11 +8,15 @@
   </PropertyGroup>
 
     <ItemGroup>
-      <PackageReference Include="serilog" Version="2.11.0" />
-      <PackageReference Include="Serilog.Sinks.Console" Version="4.0.1" />
+      <PackageReference Include="serilog" Version="2.12.0" />
```

Comment:
Latest in serilog seems to be 3.10 ? do we want to use that ?

Reply:
all other package versions are dev version. Hence not upgraded